components/content/Icon/Icon.css

```diff
@@ -0,0 +1,41 @@
+diamond-icon {
+  align-items: baseline;
+  display: inline-flex;
+
+  &::part(icon) {
+    align-items: center;
+    display: flex;
+    height: 1em;
+    justify-content: center;
+    position: relative;
+    top: var(--diamond-icon-baseline-adjust);
+    width: var(--diamond-icon-size);
+  }
+
+  svg {
+    aspect-ratio: 1 / 1;
+    display: block;
+    flex: 0 0 var(--diamond-icon-size);
+    height: var(--diamond-icon-size);
+    min-width: var(--diamond-icon-size);
+    width: var(--diamond-icon-size);
+  }
+
+  &[shape='circle'] {
+    &::part(icon)::before {
+      background-color: var(--diamond-theme-icon-color);
```

Comment:
Should `--diamond-theme-icon-color` apply for default icons too?

Reply:
Yes! missed that